dscache/capture_log.go

```diff
@@ -0,0 +1,96 @@
+package dscache
+
+import (
+	"context"
+	"fmt"
+
+	golog "github.com/ipfs/go-log"
+)
+
+// captureLog can replace this package's logger so that it counts calls instead of actually logging.
+// To be used only from tests
```

Comment:
If this is only for tests rename this file to `capture_log_test.go` to prevent anything in here from making it into the compiled binary

Reply:
Done.